.github/workflows/quic_matrix.json

```diff
@@ -1,5 +1,7 @@
 [
     { "env": "azure", "os": "ubuntu-20.04", "arch": "x64", "tls": "openssl",  "io": "epoll" },
+    { "env": "lab", "os": "ubuntu-22.04", "arch": "x64", "tls": "openssl3",  "io": "epoll" },
+    { "env": "lab", "os": "ubuntu-22.04", "arch": "x64", "tls": "openssl3",  "io": "xdp" },
```

Comment:
Please add to the end of the list with the other lab. Also, please fix formatting offset so they line up.